.idea/workspace.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file shouldn't be committed

Reply:
you could add it to the gitignore

Reply:
hm I've tried adding it multiple times with git rm, not sure why it's being so persistent. Got it removed earlier but it keeps coming back. Will look into this.